src/main/java/org/zalando/nakadi/domain/ItemsWrapper.java

```diff
@@ -5,15 +5,15 @@
 import java.util.List;
 
 @Immutable
-public class SubscriptionListWrapper {
+public class ItemsWrapper<T> {
```

Comment:
As you see in my PR I also changed that to fulfill the needs of subscriptions list endpoint  https://github.com/zalando/nakadi/pull/396/files#diff-833e3b7bf1ff84c025e9890baddd9ee8R17
So I suggest that you create a new class instead of renaming SubscriptionListWrapper